cmd/execution/main.go

```diff
@@ -134,11 +137,12 @@ func main() {
 			flags.BoolVar(&pauseExecution, "pause-execution", false, "pause the execution. when set to true, no block will be executed, but still be able to serve queries")
 			flags.BoolVar(&enableBlockDataUpload, "enable-blockdata-upload", false, "enable uploading block data to GCP Bucket")
```

Comment:
"enable uploading block data to GCP Bucket" -> "enable uploading block data to various buckets"? Or something similar, as its not longer GCP